EXILED.props

```diff
@@ -15,6 +15,7 @@
 
   <PropertyGroup>
     <!-- This is the global version and is used for all projects that don't have a version -->
+
```

Comment:
unnecessary change

Reply:
> unnecessary change

Idk how it's appears but it's not really a problem ^^

Reply:
I go look after but i don't think i have commit it probably a merge of exiled dev have make this ^^'